va/va_enc_vp9.h

```diff
@@ -506,6 +510,8 @@ typedef struct  _VAEncPictureParameterBufferVP9
      */
     uint32_t    skip_frames_size;
 
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t                __reserved[VA_PADDING_LOW];
```

Comment:
It is similar to HEVC.
It will be better to reserve more bytes for VP9. 
The EncPictureParameterBufferVP9 can be VA_PADDING_MEDIUM.